src/app/legacy/containers/PageHandlers/withContexts.jsx

```diff
@@ -41,6 +41,7 @@ const WithContexts = Component => {
         >
           <RequestContextProvider
             bbcOrigin={bbcOrigin}
+            derivedPageType={path(['metadata', 'type'], pageData)}
```

Comment:
Would this be an issue type/prop wise as path returns 'undefined' if it can't retrieve the type? Also is withThemProvider as it's expecting a string | null?

Reply:
I'll update the type and remove the `?` from it so that it will always be either `string` or `null`. The value is defaulted to `null` in `RequestContext`, so it should always be one or the other.